app/views/finders/_taxon_facet.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="app-taxonomy-select">
+  <%= render "govuk_publishing_components/components/select", {
+    id: 'top_level_taxon',
+    label: "Filter by topic",
+    options: taxon_facet.topics
+  }
+  %>
+  <div class="js-required govuk-form-group gem-c-select">
+    <label class="govuk-label" for="second_level_taxon">
+     Filter by sub-topic
+    </label>
+    <select class="govuk-select" id="second_level_taxon" name="second_level_taxon">
+      <% taxon_facet.sub_topics.each do |topic| %>
+        <option
+          data-topic-parent="<%= topic[:parent] %>"
+          value="<%= topic[:value] %>"
+          <% if topic[:selected] %>selected<% end %>>
+          <%= topic[:text] %>
+        </option>
+      <% end %>
+    </select>
```

Comment:
@andysellick FYI I've not used the component here as I need this attribute on the options: `data-topic-parent="<%= topic[:parent] %>"`. Would be interested in your thoughts!

Reply:
@bilbof totally reasonable that you would do this, but we can add that option to the component for you. Let's talk offline.

Reply:
@bilbof this option has now been added (as you know). Here's [the documentation](https://govuk-publishing-components.herokuapp.com/component-guide/select/with_data_attributes) for doing it.